libraries/provider_rack_user.rb

```diff
@@ -26,11 +27,21 @@ def whyrun_supported?
           end
         end
 
+        # default rack password to '*'
+        rack_password = '*'
```

Comment:
Had a look and a discussion with Julien around this and it's out of the scope of what this PR is trying to resolve. I have raised it as a separate issue here: #11